Feature/paypal refund

daniel-rose opened the pull request.

daniel-rose committed:
Update composer.json

daniel-rose committed:
Update composer.json

daniel-rose committed:
- refactoring
- remove unused stuff
- ...

julianhyatt commented on src/FondOfSpryker/Zed/OrderApi/Business/Model/Validator/OrderApiValidator.php:
```diff
@@ -28,7 +23,7 @@ public function validate(ApiDataTransfer $apiDataTransfer)
      * @return array
      */
-    protected function assertRequiredField(array $data, $field, array $errors)
+    protected function assertRequiredField(array $data, $field, array $errors): array
```
unused function

sourcecube commented on src/FondOfSpryker/Zed/OrderApi/Business/Model/OrderApi.php:
```diff
@@ -2,108 +2,196 @@
-            throw new EntityNotSavedException('Could not save order: ' . implode(',', $errors));
+        if ($orderTransfer === null) {
+            throw new EntityNotFoundException(
+                'Could not found order.',
```
Consider lingo change to:

1. Order not found. 
or
2. Couldn't find order. / Could not find order.

daniel-rose committed:
- remove unused method
- lingo
- add required dependencies to composer.json

daniel-rose committed:
Use github actions instead of travis (#2)

- add ci stuff
- add workflow files
- cleanup dependencies
- add phpcs fixes
- ...
